shared/src/commonMain/kotlin/com/karumi/gallery/data/local/LocalPhotosDataSource.kt

```diff
@@ -0,0 +1,35 @@
+package com.karumi.gallery.data.local
+
+import com.karumi.gallery.data.local.LocalGalleryDb.photosQueries
+import com.karumi.gallery.domain.TTLCache
+import com.karumi.gallery.model.PhotoShot
+import com.karumi.gallery.model.Photos
+
+class LocalPhotosDataSource(private val cache: TTLCache) {
+
+  fun insert(photos: Photos) {
+    photos.map(::insert)
```

Comment:
Not a `map` but a `forEach`

```suggestion
    photos.forEach(::insert)
```

Reply:
😅